src/langapi/Makefile

```diff
@@ -1,4 +1,4 @@
-SRC = mode.cpp language_ui.cpp languages.cpp language_util.cpp
```

Comment:
Why is this file being removed - to me it seems to be an essential place to store the virtual table?!